.github/workflows/security.yaml

```
@@ -0,0 +1,29 @@
+---
+name: security
+
+on:
+  push:
+    branches: [main]
+  release:
+    types: [published]
+  pull_request:
+    branches: [main]
+  schedule:
+    - cron: '0 8 * * 1' # Monday - 8am - UTC
+  workflow_dispatch:
+
+jobs:
+  build:
+    permissions:
+      contents: read
+      packages: write
+    uses: scality/osis/.github/workflows/docker-build.yaml@v1
```

if specifying the local workflow, use a local path.
```suggestion
    uses: ./.github//workflows/docker-build.yaml
```